debian/control

```diff
@@ -442,3 +452,15 @@ Description: Userspace out-of-memory (OOM) killer
  systemd-oomd is a system service that uses cgroups-v2 and
  pressure stall information (PSI) to monitor and take action on
  processes before an OOM occurs in kernel space.
+
+Package: systemd-repart
+Section: admin
+Architecture: linux-any
```

Comment:
Here you should add i386, or just replace with amd64 arm64 i386 like above

Reply:
https://github.com/pop-os/systemd/pull/26#issuecomment-1367648109